standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -266,12 +275,12 @@ private void startLocalSegmentStore(int segmentStoreId) throws Exception {
                         .with(ReadIndexConfig.CACHE_POLICY_MAX_SIZE, 128 * 1024 * 1024L))
                 .include(AutoScalerConfig.builder()
                         .with(AutoScalerConfig.CONTROLLER_URI, "tcp://localhost:" + controllerPorts[0])
```

Comment:
The auto-scaler's controller-uri, shouldn't it be using a different uri scheme depending on `enableTls`?

Reply:
Fixed. Choosing scheme based on config setting.